frontend/ui/network/manager.lua

```diff
@@ -995,6 +995,23 @@ function NetworkMgr:setWirelessBackend(name, options)
     require("ui/network/"..name).init(self, options)
 end
 
+function NetworkMgr.decodeSsid(text)
```

Comment:
Forgot to say, but I'd make that decodeSSID, unless @NiLuJe has a different opinion.

Reply:
There is precedent for doing it like this, with `Pdf`, `Html` and `Png` in various places.

Reply:
But consider the eldritch horrors of `Ppdf`, `Hhtml` and `Ppng`. ;-P (Incidentally it also says Utf8 a couple lines below.)

Reply:
You mean like [`getBookUuid`](https://github.com/koreader/koreader/blob/1ef7821b66a8356510adb4d94bdedc941c356237/plugins/calibre.koplugin/metadata.lua#L164) 😉?

All joking aside, I don't particularly care either way, so let's see if @NiLuJe has any strong opinions about this.